Source/WebCore/loader/SubframeLoader.cpp

```diff
@@ -128,10 +128,11 @@ bool FrameLoader::SubframeLoader::resourceWillUsePlugin(const String& url, const
     return shouldUsePlugin(completedURL, mimeType, false, useFallback);
 }
 
-bool FrameLoader::SubframeLoader::pluginIsLoadable(const URL& url)
+bool FrameLoader::SubframeLoader::pluginIsLoadable(const URL& url, const HTMLPlugInImageElement& ownerElement, const String& mimeType) const
 {
     if (RefPtr document = m_frame->document()) {
-        if (document->isSandboxed(SandboxFlag::Plugins))
+        bool isFullMainFramePlugin = m_frame->isMainFrame() && dynamicDowncast<PluginDocument>(m_frame->document());
```

Comment:
I think `is<>` is okay here, since you're just checking if the type is `PluginDocument`
```suggestion
        bool isFullMainFramePlugin = m_frame->isMainFrame() && is<PluginDocument>(m_frame->document());
```

Reply:
Good catch, I'll change to `is<>`